source/daplink/usb2uart/usbd_user_cdc_acm.c

```diff
@@ -85,7 +85,11 @@ int32_t USBD_CDC_ACM_PortSetLineCoding(CDC_LINE_CODING *line_coding)
     UART_Config.DataBits    = (UART_DataBits) line_coding->bDataBits;
     UART_Config.Parity      = (UART_Parity)   line_coding->bParityType;
     UART_Config.StopBits    = (UART_StopBits) line_coding->bCharFormat;
+#if defined(BOARD_VBLUNO51) || defined(BOARD_VBLUNO52)
+    UART_Config.FlowControl = UART_FLOW_CONTROL_RTS_CTS;
+#else
     UART_Config.FlowControl = UART_FLOW_CONTROL_NONE;
+#endif    
```

Comment:
I understand, but as more and more boards use flow control this code will get bigger and bigger
```
#if defined(BOARD_VBLUNO51) || defined(BOARD_VBLUNO52) || ... || defined(BoardN)
```
If this is a runtime changeable option and is set in board specific code the all boards can use the same code without defines.